resources/definitions/Federation.graphql

```diff
@@ -47,6 +48,8 @@ directive @interfaceObject on OBJECT
 directive @authenticated on FIELD_DEFINITION | OBJECT | INTERFACE | SCALAR | ENUM
 directive @requiresScopes(scopes: [[federation__Scope!]!]!) on FIELD_DEFINITION | OBJECT | INTERFACE | SCALAR | ENUM
 directive @policy(policies: [[federation__Policy!]!]!) on FIELD_DEFINITION | OBJECT | INTERFACE | SCALAR | ENUM
+directive @context(name: String!) repeatable on INTERFACE | OBJECT | UNION
+directive @fromContext(field: ContextFieldValue) on ARGUMENT_DEFINITION
```

Comment:
Could you please clarify where the ContextFieldValue type is defined? It currently shows an error for every schema when the Federation library is enabled.
![image](https://github.com/user-attachments/assets/bf6fbd01-1229-4421-a082-db2d20ec1d9b)

Reply:
In the docs I see that the directive is defined as
```
scalar ContextFieldValue;

directive @fromContext(field: ContextFieldValue) on ARGUMENT_DEFINITION;
```

Reply:
It seems that `federation__ContextFieldValue` is the correct name, but then I don't understand how it works 🤔

Reply:
The scalar names can be either one. Users won't need to know or use them directly, it is a way to declare the syntax of the string parsing which is different from other scalars like FieldSelection. 

We can update either name, to have the federation__ prefix or not. The other scalars do not today so that might be more consistent